src/k8s/objects/v2v/vmware/conversionPod.js

```diff
@@ -2,7 +2,14 @@ import { getKubevirtV2vConversionContainerImage, getV2vImagePullPolicy } from '.
 import { PodModel } from '../../../../models';
 import { CONVERSION_BASE_NAME, CONVERSION_GENERATE_NAME } from '../../../requests/v2v';
 
-export const buildConversionPod = ({ volumes, volumeMounts, namespace, serviceAccountName, secretName }) => ({
+export const buildConversionPod = ({
+  volumes,
+  volumeMounts,
+  namespace,
+  serviceAccountName,
+  secretName,
+  kubevirtVmwareConfigMap,
```

Comment:
can we please rather pass the `image` & `imagePullPolicy` to lower down the number of dependencies/imports

Reply:
I wanted to have flexibility and not pass all the params explicitly via the hierarchy. I can leverage destructuring

Reply:
done